### DISCUSSION TITLE
About Dialogue Character Name

### DISCUSSION THREAD

**Discussion by chinosk6** [chinosk6/scsp-localify#14] [General]
Searching a way to get character names from commu `key` value... #9

**Comment by ToastyBuns3939** [comment]
For some reason, they also store the character names with their corresponding dialogue key value in the audio files. But there has to be a better way to grab them other than to mess with criware audio files.

![image](https://github.com/chinosk6/scsp-localify/assets/93896872/a05e9439-a2a5-4c02-acd6-534221ea1f08)
